js/controller.js

```diff
@@ -1209,14 +1217,21 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, this.state.currentVideoId, currentTrack);
     },
 
-    onSeeked: function() {
+    onSeeked: function(event, time) {
       this.state.seeking = false;
       if (this.state.queuedPlayheadUpdate) {
         OO.log('popping queued update');
         this.skin.updatePlayhead.apply(this.skin, this.state.queuedPlayheadUpdate).then(() => {
           this.state.queuedPlayheadUpdate = null;
         }).catch(() => {OO.log('onSeeked: Could not set new state for skin')});
       }
+      if (this.state.cast.isReceiver) {
+        this.skin.updatePlayhead(time);
```

Comment:
@nzaitcev 
Just a heads up, this is already handled on the receiver in an non-elegant way(https://github.com/ooyala/chromecast-sample-receiver/blob/master/lib/src/app/player.js#L512)
It would be great to remove that part on the receiver and keep your change, which is the better solution.